packages/client/src/components/FilteringArea/FilteringSection.js

```diff
@@ -25,8 +30,11 @@ export const FilteringSection = ({
     showAllButton.style.borderColor = 'rgba(0, 0, 0, 0.2)';
     if (e.target.checked) {
       setSelectedValue(selectedValue.filter((item) => item !== e.target.value));
+      e.target.style.borderColor = '#8c5c02';
```

Comment:
This should be done by setting some state for this component instead and toggling a CSS class.
This pattern is very not-React.